global-elements/heroes.md

```diff
@@ -32,7 +32,7 @@ Heroes function as a primary focal point on a page, often used to introduce a co
 </div> 
```

Comment:
This would seem to be an appropriate PR to update this block, as well. We're going to have these issues on most of these pages, though, so maye we should batch them all in one PR later?